src/types/global.d.ts

```diff
@@ -0,0 +1,5 @@
+export {};
+
+declare global {
+  let use: (dbName: string) => void;
+}
```

Comment:
I’m guessing the next step here is to figure out how to create a .d.ts file for mongosh environments? Probably not something for this PR, though :slightly_smiling_face: